zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -20,6 +20,7 @@
 public class Configuration {
 
     private static final String DEFAULT_WEBSITE_URL = "http://localhost:8080";
+    private static final String DEFAULT_CACHE_API_URL = "http://localhost:13580";
```

Comment:
Wrong port, it should be `13850`. But that wouldn't be right either, because that's the non-local port, and the rest of the URLs in this config seem to be using the local port. So I think this should be 29100.

In addition to that, the full name is Legacy Cache API, so we should probably rename all occurrences of "Cache API" to "Legacy Cache API", as that's what we've been referring to in other repos, like the proxy.

Reply:
Resolved this